homeassistant/components/ruuvi_gateway/manifest.json

```diff
@@ -4,6 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/ruuvi_gateway",
   "codeowners": ["@akx"],
+  "dependencies": ["bluetooth"],
```

Comment:
Shouldn’t this be `bluetooth_adapters`

Reply:
This one is actually providing a bluetooth adapter so its not a consumer of `bluetooth_adapters`